packages/tools/src/tools/displayTools/Labelmap/labelmapConfig.ts

```diff
@@ -4,6 +4,7 @@ const defaultLabelmapConfig: LabelmapConfig = {
   renderOutline: true,
   outlineWidthActive: 3,
   outlineWidthInactive: 2,
+  activeSegmentOutlineWidth: 4,
```

Comment:
Should these be percentages, such that this is the displayed pixel width in actual pixels on screen?

Reply:
the unit is pixels, I guess you are right, but It needs to change everything maybe for 2.x